massa-consensus-exports/src/tools.rs

```diff
@@ -0,0 +1,62 @@
+use std::collections::HashMap;
```

Comment:
These are tools meant to be only used in tests, I'd rename/move the file to make that clear

Reply:
I think that's a good point, best to move those to the worker where the tests are.

Reply:
Actually, one of these functions is also used in an implementation of ConsensusConfig (the `impl<&Path>`) that cannot be moved easily. We should first think about _"Where we should put the common configs tools and implementations"_. I'm open to any proposals!!